volatility3/framework/plugins/windows/modules.py

```diff
@@ -127,6 +127,32 @@ def _generator(self):
                 file_output,
             )
 
+    @classmethod
+    def get_kernel_space_start(cls, context, layer_name: str, module_name: str) -> int:
```

Comment:
I might be tempted to swap these around, and also, I believe that modules come with an associated layer.  If there's reason to want a different layer to the module, then this is fine, but otherwise I'd just drop the `layer_name` parameter and uses the module's `layer_name` instead?